rust/xaynet-server/src/state_machine/mod.rs

```diff
@@ -229,9 +236,11 @@ type StateMachineInitializationResult<T> = Result<T, StateMachineInitializationE
 pub enum StateMachineInitializationError {
     #[error("redis request failed: {0}")]
     Redis(#[from] RedisError),
+    #[error(transparent)]
+    Store(#[from] StorageError),
     #[error("failed to initialize crypto library")]
     CryptoInit,
-    #[error("failed to fetch global model: {0}")]
+    #[error("{0}")]
```

Comment:
why removing the first part of the error description? It seems useful.
If the concern is that it's somewhat redundant with the inner String, I think it's fine. I'd rather have slightly redundant information than lose a bit of it

Reply:
previously there was no distinction between io error and the model does not exist
the io error moved to L240
and this error is just for the case that the model does not exist
I can add `failed to fetch global model:` to the `StorageError` wdyt?

Reply:
ahh ok I understand why you removed it then. +1 for adding this to `StorageError`